src/Redux/fireRequest.tsx

```diff
@@ -170,9 +170,15 @@ export const fireRequest = (
 
           // 5xx Errors
           if (error.response.status >= 500 && error.response.status <= 599) {
-            Notification.Error({
-              msg: "Something went wrong...!",
-            });
+            if (error.response.data && error.response.data.detail) {
+              Notification.Error({
+                msg: error.response.data.detail,
+              });
+            } else {
+              Notification.Error({
+                msg: "Something went wrong...!",
+              });
+            }
```

Comment:
But issue was to check it for the internal server errors, i.e. 5xx errors as well. Right?

Reply:
@jainvedant392 

https://github.com/coronasafe/care_fe/pull/6856/files/326338e00b99ccd72d3e5ad8c9122f53032d7377#diff-e2c5c73d556d02503d5adc253828ddf5bad94f6fb9fba25df333d5ca6b9b5c56R152

Just setting the upper limit to `< 600` instead of `< 400` should work right?

Because it looks like the 4xx code block handles the error the same as what you've done in 5xx code block. We could just remove the 5xx code block and tweak the 4xx code block to cover both 4xx and 5xx right?

Reply:
Yeah I think that would be right. I'll make the changes.